### ISSUE TITLE
Bug: Setting `concurrency=1` in OpenAI model params immediately crashes process with no error message on first call

### ISSUE THREAD

**Issue by kryptoklob** [hwchase17/langchainjs#418]
This one was quite the bugger to find. Because of GPT4's rate limit, I set the `concurrency` param to `1` when instantiating the `OpenAI` model as you can see below:

```
const model = new OpenAI({
  openAIApiKey: <redacted>,
  modelName: 'gpt-4',
  temperature: 0.7,
  verbose: true,
  concurrency: 1,
  callbackManager: callbackManager,
  maxTokens: 1000,
  maxRetries: 5
})
```

Even with explicit callbacks, this causes the program to exit *immediately* upon making its first call. Values higher than one (including `2`) work fine.

**Comment by nfcampos** [comment]
@kryptoklob I'm looking into this, thanks for reporting

**Comment by nfcampos** [comment]
This has been fixed in 0.0.39
